src/components/PathCard.tsx

```diff
@@ -133,8 +143,8 @@ export default function PathCard({
 		</View>
 	);
 
-	return network.protocol === NetworkProtocols.SUBSTRATE ||
-		network.protocol === NetworkProtocols.UNKNOWN
+	return isSubstrateNetworkParams(networkParams) ||
+		isUnknownNetworkParams(networkParams)
```

Comment:
This is equivalent to `return isUnknownNetworkParams(networkParams)`, see below.